src/raftstore/store/store.rs

```diff
@@ -528,7 +558,7 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             return Ok(());
         }
 
-        if !try!(self.check_target_peer_valid(region_id, msg.get_to_peer())) {
+        if !try!(self.maybe_create_peer(region_id, &msg)) {
```

Comment:
Considering the context and its function, IMO the original name is a better name.